models/gemini/main.py

```diff
@@ -1,7 +1,5 @@
-import grpc.experimental.gevent
 from dify_plugin import Plugin, DifyPluginEnv
 
-grpc.experimental.gevent.init_gevent()
```

Comment:
I’m curious about the reasoning behind this change. Was the removal of `grpc.experimental.gevent.init_gevent()` intentional, or might it have been unintentional?